Languages/po/de.po

```diff
@@ -3100,6 +3119,11 @@ msgid ""
 "\n"
 "If unsure, leave this unchecked."
 msgstr ""
+"Jegliche Anfrage von der CPU das EFB zu lesen oder schreiben wird ignoriert.\n"
+"Die Emulationsgeschwindigkeit wird in machen Spielen verbessert. Es können "
```

Comment:
"Die Emulationsgeschwindigkeit wird in manchen Spielen verbessert. Es können "
"jedoch Spielfunktionen und grafische Effekte deaktiviert werden.\n"